padguide2/padguide2.py

```diff
@@ -2456,6 +2516,11 @@ def __init__(self, monster: PgMonster, monster_group: NamedMonsterGroup, prefixe
         # This stuff is important for nickname generation
         self.group_basenames = monster_group.basenames
         self.prefixes = prefixes
+        
+        # Pantheon
+        series = monster.series
+        if series:
+            self.series = series.name

```

Comment:
Avoid conditional properties on objects. You can rewrite this to:
`self.series = monster.series.name if monster.series else None`

Reply:
ok fixed